scripts/skill_auditor/test_skill_auditor.py

```
@@ -0,0 +1,323 @@
+        "forbidden_files": [],
+        "yaml_delimiters": 2,
+        # Missing: has_name, has_description, line_count, etc.
+    }
+
+    # This should raise ValueError
+    try:
+        # Assume we'll create a validate_metrics_structure function
+        from validation import validate_metrics_structure
+
+        validate_metrics_structure(incomplete_metrics)
+        assert False, "Should have raised ValueError"
```

Ruff (0.14.5)

Assertion always fails, replace with `pytest.fail()`

(PT015)